Makefile

```
@@ -0,0 +1,44 @@
+ARCH ?= amd64
+KEY ?= ""
+
+VERSION ?= $(shell git describe --abbrev=0 --tags)
+LOCAL_VERSION ?= $(shell git rev-parse --short HEAD)
+
+######################################################################################
+
+.PHONY: help
+help: ## Display this help information
```

Would it make sense to put back some of the venv make targets for simplicity?
```suggestion
help: ## Display this help information
create-venv:
	python -m venv .venv
requirements: ##or dependencies or something
        python -m pip install .
```
Alternatively, some instructions to set up a virtual environment in the README, because at least one of the make targets won't work without installing at least some of the dependencies in `pyproject.toml`

---

I think not, at least for now?
I am all for adding more instructions to the README instead of adding these to make targets though.


The one edge case for managing our python dependencies in the make targets could be for the `gen-python` target. If that is something that comes up in the future I'm down add something like this to that specific target.